.circleci/generate.sh

```diff
@@ -184,6 +190,14 @@ fi
 
 # add new or modified tests to the sets of tests to be repeated
 if (!($all)); then
+  # Sanity check that the referenced branch exists
+  if ! git show ${BASE_BRANCH} -- >&/dev/null; then
+    echo -e "\n\nUnknown base branch: ${BASE_BRANCH}. Unable to detect changed tests.\n"
+    echo    "If you wish to detect changed tests, please use the '-b' flag to choose existing"
```

Comment:
Since we are making the detection of tests mandatory, I think that the `If you wish to detect changed tests,` part of the message is not needed anymore, or it should be rephrased. Also, it should probably be `use the '-b' option` instead of `please use the '-b' flag`.

Reply:
Fixed in latest commit